packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -48,6 +49,12 @@ export async function processOutboundDocumentQueryResps({
     const duration = elapsedTimeFromNow(docQueryStartedAt);
 
     const docRefs = results.map(toDocumentReference).flat();
+    const contentTypes = docRefs.map(docRef => {
+      if (!docRef.contentType) return "unknown";
+
+      return docRef.contentType;
+    });
+    const contentTypeCounts = getDocumentReferenceContentTypeCounts(contentTypes);
```

Comment:
Can we pass `docRefs` to that `getDocumentReferenceContentTypeCounts`, so we minimize analytics-related code on the regular app logic?